### PR TITLE
Added footer in 404 page

### LINKED ISSUE #211: Add footer to error page
Currently, all the pages including the homepage and meeting-details page have a header and a footer. But, the 404 error page doesn't have a footer at all, which is inconsistent and looks incomplete. A similar footer should be added to the 404-error page as well.

I would like to work on this @darknao @t0xic0der

### PR DESCRIPTION
**Pull request by subhangi2731**
Closes #211 

Added footer in 404-error page.

### CODE REVIEW

**Comment by subhangi2731** [comment]
@darknao Can you share a screenshot of the footer being off the bottom? In my case, the 404 page footer is still at the very bottom of the page in all window sizes. 
Regarding the width, as you may notice, the 404 page's content takes up the whole page in comparison to meeting-details page where a box within the page shows the details and contains the footer.

**Comment by darknao** [comment]
image

**Comment by subhangi2731** [comment]
In my case, the footer looks as usual
Screenshot from 2021-11-04 08-23-08

**Review by t0xic0der — CHANGES_REQUESTED** [review]
Same for me, I guess.

image

@subhangi2731, could you check if there's an inbuilt footer component packed in Bootstrap 5?

**Comment by subhangi2731** [comment]
@t0xic0der @darknao With the help of the in-built bootstrap I can add a footer in all the pages but this is a part of this issue #170. So if you can assign me that issue both will be covered under the same PR.
Screenshot from 2021-11-04 08-51-38

**Comment by subhangi2731** [comment]
@darknao I have updated the changes please review

**Comment by subhangi2731** [comment]
@t0xic0der @darknao please review my updated PR

**Comment by darknao** [comment]
This is still not working.
Check this bootstrap example for reference : https://getbootstrap.com/docs/5.0/examples/sticky-footer-navbar/
Note that you need to use flexbox to make it work like that.

But there is several way to solve this issue.
If you look at others pages, you'll notice the footer is actually a card footer.
Why not just use this same logic here, and move everything in a card, like in the main page?

**Comment by subhangi2731** [comment]
@darknao okay I will do this and update the PR

**Review by t0xic0der — CHANGES_REQUESTED** [review]
Why the long padding at the top?

image

**Comment by subhangi2731** [comment]
@t0xic0der sorry will correct the padding other than this is everything okay?

**Comment by t0xic0der** [comment]
@subhangi2731, I'd suggest that you correct the padding first and then maybe, I'll give it another round of review to see if there's anything else that needs changing.

**Comment by subhangi2731** [comment]
@t0xic0der sure

**Review by t0xic0der — APPROVED** [review]
LGTM.

Merging.

**Review by t0xic0der — CHANGES_REQUESTED** [review]
Please ignore https://github.com/fedora-infra/mote/pull/212#pullrequestreview-819774086. Changes are still needed.

**Comment by subhangi2731** [comment]
@t0xic0der @darknao I corrected the long padding, kept all the contents in the card component, and then also brought the footer in correct alignment and now it's totally down touching the end of the page. Please check and let me know

**Comment by subhangi2731** [comment]
Screenshot from 2021-12-01 20-01-35

**Comment by subhangi2731** [comment]
footer alignment

**Comment by subhangi2731** [comment]
Screenshot from 2021-12-01 20-21-09

**Comment by subhangi2731** [comment]
Screenshot from 2021-12-01 20-23-03

File: src/mote/templates/e404page.html [hunk 1]
```diff
@@ -36,21 +36,48 @@
                 </div>
             </div>
         </nav>
+        <div class="container-lg" style="padding-top: 20px;">
+            <div class="card mt-2 mb-2">
+                <div class="card-body text-center">
+                    <img src="{{ url_for('static', filename='imgs/404_not_found.svg') }}" style="width: 40%; height: 40%;" />
```
Comment (darknao) [inline src/mote/templates/e404page.html:42]: It works fine on Chrome, but have some extra padding at the bottom on Firefox.
One way to fix that is by removing the height css attribute on the 404 image (which is not really needed anyway):
```suggestion
                    <img src="{{ url_for('static', filename='imgs/404_not_found.svg') }}" style="width: 40%;" />
```

**Comment by subhangi2731** [comment]
@darknao sure

**Comment by subhangi2731** [comment]
@darknao please check and let me know

**Review by darknao — APPROVED** [review]
:+1:

**Review by t0xic0der — APPROVED** [review]
Made one small change to bring the card segment to the center of the page.

**Comment by subhangi2731** [comment]
@t0xic0der great

**Pull request merged**